### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][IntelliJ][Wipro]: Keyboard focus is not visible on "Help" button in Run web app window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1437]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app.
4. "Run on web app" dialog will open.
5. Tab to and select "Use Existing web app" radio button.
6. Tab to Help (?) button and hit Space bar and observe .

**Note:** Same issue is observed in Run on Web app-Create new web app for the same button.

**Expected Result:**
Keyboard focus is should visible on "Help" button in Run web app window.

**Actual Result:**
Keyboard focus is not visible on "Help" button in Run web app window.

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![no keyboard focus on help button](https://user-images.githubusercontent.com/22956130/35727533-40c5dcaa-07bd-11e8-8431-c801bc215c2f.png)

**Comment by Eskibear** [comment]
> Platform:
> Eclipse Version:
> Eclipse Java EE IDE for Web Developers.
> Version: Oxygen.2 Release (4.7.2)
> Build id: 20171218-0600

Seems wrong.

**Comment by Eskibear** [comment]
duplicated with https://github.com/Microsoft/azure-tools-for-java/issues/1456

**Comment by WiproPSAccTest** [comment]
Verified in 2017.3.5 Community.
Issue still exists

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. Very low usage for this action.
* **Quality of experience**: not impacted. Users can view guidance in the [docs.microsoft.com](https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable#deploy-your-web-app-to-azure). 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749903
